### DISCUSSION TITLE
Question - what is the best approach, to modify existing Zenodo record?

### DISCUSSION THREAD

**Discussion by rkrug** [eblondel/zen4R#105] [Q&A] [answered]
Hi

I would like to deposit a large amount of data in many deposits. For the moment, the metadata can be filled in, but there will be changes in the near future to the metadata, but not to the actual deposited data. 

What is the best approach ti di this? Is ist sufficient to save the DOIs of the deposits? Is there anything else I should save locally to make the workflow of editing metadata more straight forward? I would assume that searching for each deposit by DOI will be quite slow?

Thanks,

Rainer

**Comment by eblondel** [comment, marked as answer]
@rkrug I'm moving this as 'discussion'.
In Zenodo, the DOI issuance relates to data. This means that after publishing a record (what makes the DOI alive), the metadata can be edited. You can then fetch a published record, modify the metadata and republish it. However, you can't modify deposited data on an existing published record (DOI binds to data).
On the contrary, if you create a new version of the record (with a new DOI), you must change the data (at least part of it). This could be a new version of your data file, additional files, etc. If you don't change the deposited data, you will not be allowed to create a new version.

In Zenodo, search deposits is quite efficient. You can then fetch a deposit by its DOI, edit the record (for metadata only) and re-submit it.

**Comment by rkrug** [comment]
Sorry - have overlooked the Discussion tab...

So storing the DOIs locally and retrieving the records is an appropriate way - sounds easy.

**Reply by eblondel** [reply]
Yes, you can store your DOIs once you deposited your draft records, not necessarily published them, since when you create a draft deposit you will get the reserved DOI.
